cea/scripts.yml

```
@@ -273,6 +273,56 @@ Life Cycle Analysis:
       - [get_zone_geometry]
       - [get_building_supply]
 
+  - name: hourly-operational-emission
+    label: Hourly Operational Emission
+    short_description: Generate a detailed hourly operational emission timeline of a typical year.
+    description: This Feature generates operational emission for heating, cooling, hot water 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)